### PR TITLE
date shortcut

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
* Target version: master 
grafik

nextcloud/nextcloud-vue#1419

### CODE REVIEW

File: src/components/card/CardSidebarTabDetails.vue [hunk 1]
```diff
@@ -176,6 +177,23 @@ export default {
...
+			shortcuts: [
+				{
+					text: 'Today',
+					onClick() {
+						const date = new Date()
+						return date
+					},
+				},
+				{
+					text: 'Tomorrow',
```
Comment (juliushaertl) [inline src/components/card/CardSidebarTabDetails.vue:189]: Shouldn't we then add more shortcuts here, like next week, next month?

File: src/components/Controls.vue [hunk 2]
```diff
@@ -1,416 +0,0 @@
-<!--
```
Comment (juliushaertl) [inline src/components/Controls.vue:1]: Why is this file being deleted?

File: src/components/card/CardSidebarTabDetails.vue (hunk 1 above)
Comment (juliushaertl) [reply src/components/card/CardSidebarTabDetails.vue:189]: Please also make the labels translatable

File: src/components/card/CardSidebarTabDetails.vue [hunk 3]
```diff
@@ -176,6 +177,23 @@ export default {
 				stringify: this.stringify,
 				parse: this.parse,
 			},
+			shortcuts: [
+				{
+					text: 'Today',
+					onClick() {
+						const date = new Date()
+						return date
```
Comment (juliushaertl) [inline src/components/card/CardSidebarTabDetails.vue:185]: I'm not sure what today should ideally do but setting it to the current time will cause the file to be over due immediately.

File: src/components/card/CardSidebarTabDetails.vue [hunk 4]
```diff
@@ -176,6 +177,23 @@ export default {
...
+					},
+				},
+				{
+					text: 'Tomorrow',
+					onClick() {
+						const date = new Date()
+						date.setTime(date.getTime() + 3600 * 1000 * 24)
```
Comment (juliushaertl) [inline src/components/card/CardSidebarTabDetails.vue:192]:
```suggestion
						date.setDate(date.getDate() + 1)
```

File: src/components/card/CardSidebarTabDetails.vue [hunk 5]
```diff
@@ -316,6 +334,10 @@ export default {
 </script>
 <style lang="scss" scoped>
 
+.section-wrapper::v-deep .mx-datepicker-main .mx-datepicker-popup {
```
Comment (juliushaertl) [inline src/components/card/CardSidebarTabDetails.vue:337]:
```suggestion
.section-wrapper::v-deep .mx-datepicker-main.mx-datepicker-popup {
```

Both classes are applied on the same element

image

**Comment by jakobroehrl** [comment]
https://github.com/nextcloud/deck/pull/2710

**Pull request closed without merging**